content/scout/exceptions.md

```diff
@@ -0,0 +1,198 @@
+documents under `.vex/` in the build context, to `/var/lib/db` in the image.
+
+```dockerfile
+# syntax=docker/dockerfile:1
+
+FROM alpine
+COPY .vex/* /var/lib/db/
+```
+
+The filename of the VEX document must match the `*.vex.json` glob pattern.
+It doesn't matter where on the image's filesystem you store the file.
+
+Note that the copied files must be part of the filesystem of the final image,
+For multi-stage builds, the documents must persist in the final stage.
+
+### Attestation
+
+To attach exceptions as an attestation, you can use the `docker scout
+attestation add` CLI command.
+
+Using attestations is the recommended option for attaching exceptions to
+images. You can attach attestations to the image after it's built and pushed to
+a registry, meaning you don't need to create the VEX documents before you
+build. Additionally, having the exceptions attached to the image as
+attestations means consumers can inspect the exceptions for an image, directly
+from the registry.
+
+To attach an attestation to an image:
+
+1. Build the image and push it to a registry.
+
+   ```console
+   $ docker build --provenance=true --sbom=true --tag <IMAGE> --push .
+   ```
+
+2. Attach the exception to the image as an attestation.
+
+   ```console
+   $ docker scout attestation add \
+     --file <cve-id>.vex.json \
+     --predicate-type https://openvex.dev/ns/v0.2.0 \
+     <IMAGE>
+   ```
+
+## View exceptions
+
+The **Exceptions** page on the [Docker Scout Dashboard](https://scout.docker.com/)
+lists the exceptions for all images in your organization. Selecting a row in
+the list opens the exception side panel, which displays more information about
+the exception and where it comes from.
+
+To view all exceptions for a specific image tag:
+
+{{< tabs >}}
+{{< tab name="Docker Scout Dashboard" >}}
+
+1. Go to the [Images page](https://scout.docker.com/reports/images).
+2. Select the tag that you want to inspect.
+3. Open the **Image attachments** tab.
+
+{{< /tab >}}
+{{< tab name="Docker Desktop" >}}
+
+1. Open the **Images** view in Docker Desktop.
+2. Open the **Hub** tab.
+3. Select the tag you want to inspect.
+4. Open the **Image attachments** tab.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+## Debugging VEX documents
+
+Adding a VEX document to an image doesn't automatically validate the document.
+The `vexctl` command line tool 
+To test whether the VEX documents you create are well-formed and produce the
+expected results, you can use the `docker scout cves` command with the
+`--vex-location` flag to apply a local VEX document to an image analysis.
+
+The following examples shows how to run an image analysis with a local VEX
+document. The argument for `--vex-location` is the path of a local directory
+that contains one or more VEX documents. In this example, the current working
+directory.
+
+```console
+$ docker scout cves <IMAGE> --vex-location .
+```
+
+The output of the `docker scout cves` command should factor in any VEX
```

Comment:
Can we clarify here what "factor in" means? Need to double check with engineering, but at the very least this means that the exception CVE will be excluded from the output

Reply:
yes, I will give that as an example. I don't think we need to discuss any other statuses here, as this is about exceptions.